content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
++++ 
```

Comment:
I think the blog front matter is usually enclosed with `---`

```suggestion
---
```

Reply:
(all the `=` will need to be changed to `:` to make the front matter yaml compliant)

Reply:
Also, I think the blog system needs a slug in order to render the page correctly, something like:

```
slug: thank-you-dan-kohn
```

Then, when the other front matter issues are cleared up, the deploy preview will show up at `preview.url/blog/2021/03/19/thank-you-dan-kohn/` (or something similar)